k8s/namespaces/wa/wa-task-configuration-api/wa-task-management-api.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-configuration-api
+spec:
+  releaseName: wa-task-configuration-api
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/wa-task-configuration-api
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/wa/task-configuration-api:prod-9c626f9b
```

Comment:
if that image tag does not exit then flux will fail. One solution is to change to 'latest' and then flux will update it automatically to whatever prod tag is the latest.

Reply:
Thanks you're right I'll change to latest